docs/data/sdks/sdk-quickstart.md

```diff
@@ -572,56 +555,130 @@ Use this guide to get started with the Amplitude SDKs. Choose your target platfo
 
     Learn more available functionalities in [Android SDK](../android-kotlin/).
 
-=== "iOS"
+    --8<-- "includes/sdk-quickstart/quickstart-enforce-event-schema-intro.md"
+
+    === "Kotlin"
+
+        ```kotlin
+
+        ampli.load()
+
+        ampli.yourEventType(
+            stringProp = "Strongly typed property",
+            booleanProp = true
+        )
+
+        ```
+    === "Java"
+
+        ```java
+        Ampli.getInstance().load();
+
+        Ampli.getInstance().yourEventType(
+            YourEventType.builder()
+                .stringProp("Strongly typed property")
+                .booleanProp(true)
+                .build()
+        );
+        ```
+
+    Learn more about [Ampli Android](../android-kotlin/ampli/).
 
+=== "iOS-Legacy"
```

Comment:
Maybe this should be "iOS". until Swift is out of Beta. See other comment below.

Reply:
As discussed in team sync:
- removed "Legacy"
- move older version iOS out of Legacy SDKs section on the nav bar
- replace the banner connect as you suggested above in both quickstart and ios full doc page